### PR TITLE
AK Lidar + AK Bonnell GPR Add

### PR DESCRIPTION
**Pull request by micahjohnson150**
- Added AK lidar script for:
1. lidar dervied canopy height model from AK 
2. March Lidar flight in 2023 in AK 

- Added Bonnell AK GPR Script

### CODE REVIEW

**Comment by micahjohnson150** [comment]
Fixes #27

File: scripts/upload/add_csu_gpr_AK.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    # Upload is not able to handle the Notes col. So just remove it for now
+    modified = file.parent.joinpath(file.stem + f'_mod{file.suffix}')
+    print(f"Removing Notes Column prior to upload. Writing to {modified}")
+
+    coi = [c for c in df.columns if c != 'Notes']
+
+    # No time is a problem. Use 12 AKST == 9pm UTC
+    df['Time[HHMM]'] = '09:00'
+
```
Comment (micahjohnson150) [inline scripts/upload/add_csu_gpr_AK.py:30]: Note current preliminary file has Nans for time which is a no go.

File: scripts/upload/add_csu_gpr_AK.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    kwargs = {
+        # Keyword argument to upload depth measurements
+        'depth_is_metadata': False,
+
+        # Constant Metadata for the GPR data
+        'site_name': 'farmers loop/creamers field',
+        'observers': 'Randall Bonnell',
+        'instrument': 'pulse EKKO Pro multi-polarization 1 GHz GPR',
```
Comment (micahjohnson150) [inline scripts/upload/add_csu_gpr_AK.py:42]: Is this the right instrument?

File: scripts/upload/add_csu_gpr_AK.py (hunk 1 above)
Comment (agentrrb) [reply scripts/upload/add_csu_gpr_AK.py:30]: Entering 9:00 local should be fine.

File: scripts/upload/add_csu_gpr_AK.py (hunk 2 above)
Comment (agentrrb) [reply scripts/upload/add_csu_gpr_AK.py:42]: Tate uses a multi-polarization. I'm still in the stone age haha.

'instrument': 'pulseEkko pro 1 GHz GPR'

**Review by micah-prime — APPROVED** [review]
Looks great!

**Pull request merged**